### ISSUE TITLE
extension fails on startup

### ISSUE THREAD

**Issue by imogenkinsman** [rflagreca/vscode-clj-zprint#8]
Hi - thank you so much for making this extension, it's super helpful.

The extension is crashing for me on the latest `0.0.5`. I took a look at the extension logs and saw this:

```
[2022-09-15 15:08:36.993] [exthost] [error] Activating extension rflagreca.vscode-clj-zprint failed due to an error:
[2022-09-15 15:08:36.993] [exthost] [error] TypeError: Cannot read properties of undefined (reading 'ug')
    at $$ (/Users/imogen/.vscode/extensions/rflagreca.vscode-clj-zprint-0.0.5/extension.js:2867:108)
    at activate (/Users/imogen/.vscode/extensions/rflagreca.vscode-clj-zprint-0.0.5/extension.js:2874:222)
    at Function._callActivateOptional (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:97:17649)
    at Function._callActivate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:97:17307)
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:97:15118
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at E._activate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:8224)
    at E._waitForDepsThenActivate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:8166)
    at E._initialize (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:7530)
```

I rolled back to `0.0.4` and it formats without problems, and a co-worker was able to reproduce the same issue (with it resolving on rollback).

**Comment by rflagreca** [comment]
Hi @imogenkinsman - thanks for bringing this to my attention, I'll look in to it ASAP.  Please continue to use `v0.04` for the time being and I'll update on this issue in due course.

**Comment by rflagreca** [comment]
Hi @imogenkinsman.  

I've investigated and it turns out that zprint versions >= 1.2.3 are incompatible with the extension in its current state.  I've rolled back to version 1.2.2 for the time being and have updated the extension to v0.0.6 which is available now.  I need to do some further work to get the extension compatible with newer versions of zprint.  Thanks for reporting this issue and apologies for any inconvenience.
